pulp_docker/app/models.py

```diff
@@ -179,37 +158,22 @@ class ManifestTag(Content):
 
     name = models.CharField(max_length=255, db_index=True)
 
-    manifest = models.ForeignKey(
-        ImageManifest, null=True, related_name='manifest_tags', on_delete=models.CASCADE)
-
-    class Meta:
-        unique_together = (
-            ('name', 'manifest'),
-        )
-
-
-class ManifestListTag(Content):
-    """
-    A tagged Manifest List.
+    tagged_manifest = models.ForeignKey(
+        Manifest, null=True, related_name='tagged_manifests', on_delete=models.CASCADE)
```

Comment:
i don't think null can be set to true anymore.

Reply:
disregard this, we still need null=True because we save manifest after the tag, and the relate